SavageWorlds-Tabbed/SWTRedesign.css

```diff
@@ -4293,6 +4293,7 @@ label.sheet-ddmenu:hover {
     font-size: 1.8em;
 }
 
+.sheet-rolltemplate-info a[href^='~'],
```

Comment:
Not sure why that's there, I didn't change that file, but that is correct, regardless. So, that CSS thing should be there, just not sure why it's in this request, because it should have been part of the last one. LOL